packages/optimizer/README.md

```diff
@@ -216,14 +273,13 @@ get an intrinsic layout. For image detection to work, an optional dependency
 
 #### `minify`
 
-Minifies the generated HTML output and inlined CSS.
+Minifies the generated HTML output. Requires the following `terser` to be installed when inlined `amp-script` should be minified. Combine with [separateKeyframes](#separateKeyframes) to also minify CSS.
```

Comment:
> <pre>Requires the following `terser` </pre>

should be

<pre>Requires [`terser`](https://www.npmjs.com/package/terser)</pre>

Reply:
fixed